include/vara/Feature/FeatureModel.h

```diff
@@ -371,19 +342,47 @@ template <> struct GraphWriter<vara::feature::FeatureModel *> {
     }
   }
 
-  /// Output \a Node with custom attributes.
+  /// Output \a Feature node with custom attributes.
   void emitNode(const NodeRef Node) {
-    std::string Label = llvm::formatv(
-        "<<table align=\"center\" valign=\"middle\" border=\"0\" "
-        "cellborder=\"0\" "
-        "cellpadding=\"5\"><tr><td>{0}{1}</td></tr></"
-        "table>>",
-        DOT::EscapeString(Node->getName().str()),
-        (Node->getFeatureSourceRange()
-             ? "</td></tr><hr/><tr><td>" +
-                   DOT::EscapeString(Node->getFeatureSourceRange()->toString())
-             : ""));
-
+    std::string Label;
+    auto *F = llvm::dyn_cast<vara::feature::Feature>(Node);
+    if (F) {
+      std::stringstream CS;
+      for (const auto &C : F->constraints()) {
+        CS << "<tr><td>" << DOT::EscapeString(C->getRoot()->toHTML())
+           << "</td></tr>";
+      }
+      Label =
+          llvm::formatv(
+              "<<table align=\"center\" valign=\"middle\" border=\"0\" "
+              "cellborder=\"0\" "
+              "cellpadding=\"5\">{0}{1}{2}</table>>",
+              llvm::formatv("<tr><td><b>{0}</b></td></tr>",
+                            DOT::EscapeString(F->getName().str()))
+                  .str(),
+              CS.str(),
+              (F->getFeatureSourceRange()
+                   ? llvm::formatv("<hr/><tr><td>{0}</td></tr>",
+                                   DOT::EscapeString(
+                                       F->getFeatureSourceRange()->toString()))
+                         .str()
+                   : ""))
+              .str();
```

Comment:
```suggestion
              ;
```
I think we don't need to convert these objects to strings before using them. I'm not 100% sure so please double check :)